dictionary/README.md

```diff
@@ -2,4 +2,10 @@
 Please write a couple of sentences explaining your choice of restart
 strategy.
 
-
+For the supervision restart strategy of my dictionary application I 
```

Comment:
# -3

This is well argued, which I appreciate. However, I can't really think of any reason the the dictionary to terminate normally. The intent is that it should always be running, so I think :permanent expresses this better.

In practice, the two would likely be identical